.github/workflows/tag-version-this-repo.yml

```diff
@@ -0,0 +1,13 @@
+name: Tag Version (this repo)
```

Comment:
Please also clarify which steps are manual and which are automatic. For example steps 1-2 are manual for sure, 3-4 are automatic, but the wording is unclear for me for step 5.

Reply:
Thanks @Piedone, I'll modify the docs. I'll review those steps @BenedekFarkas - I think I may have changed to using a passed in secret after I wrote the PR steps and forgot to update.

Reply:
Add new section to readme for [Versioning, Tags and Releases](https://github.com/Lombiq/GitHub-Actions/tree/issue/OSOE-735?tab=readme-ov-file#versioning-tags-and-releases).

Reply:
I've made some updates, @Piedone you can check.

Reply:
That nicely describes what happens and how, but not why. So I'd change two things:

- Add something like this as the first paragraph under the section: "To release versions of Lombiq GitHub Actions, and allow consumers to reference both reusable workflows and composite actions in a versioned way (i.e. `@v1.0`) we employ some automation.
- Make the list a numbered one, since it describes steps in order.

Reply:
Thanks @Piedone and @BenedekFarkas, I'll make these README edits and push shortly.